episodes/collaborating-team.md

```diff
@@ -0,0 +1,256 @@
+
+1. \(S) Edit a file and commit the changes to a new branch.
+2. \(S) Open a new Pull Request from this new branch to `main`.
+        If your PR will fix an open issue on your repository,
+        include "Fixes #X" at the start of the PR description,
+        where _X_ is the issue number.
+3. \(S) Request a review from the Reviewer.
+4. \(R) Review the PR, making comments and suggesting changes as appropriate.
+        When you have finished, approve the PR or request changes, based on your review.
+5. \(S) Respond to any comments from the Reviewer and/or, if any changes were requested,
+        make those and re-request a review when you have finished.
+6. \(R) When all changes have been made to your satisfaction,
+        approve the PR, thank the Submitter for their contribution,
+        and merge the PR
+        (Or close it if the changes were made for practice but should not
+        be included in the actual lesson.)
+
+::::::::::::::::
+
+
+:::::: callout
+
+## Effective review comments are a learned skill
+
+Making effective review comments in a PR is something you will get better at with practice - here is some general guidance to help with reviewing a PR from a collaborator:
+
+- Make sure your review comments are specific and actionable.
+- Make it clear in the comment what you want changed as part of a PR and, ideally, provide a concrete suggestion.
+- Only request changes if the content is wrong or hard to understand (remember, different does not always mean better).
+- If a conversation on a PR hasn’t been resolved by one or two back-and-forth exchanges, consider scheduling a meeting to discuss further.
+- Good reviews have empathy - consider the contributor's feelings, identify positives and try not to be judgemental.
+- Aim for improved/better rather than perfect.
+
+::::::::::::::::
+
+
+## Managing Communication
+
+Having an open, publicly-visible list of all the issues with your project is a helpful way of letting people know you are aware of issues and you are working on them. This can indicate to an external audience that the project is active.
+It also provides you and your collaborators with an "at a glance" view of the state of the project, making it easier to prioritise future work.
+
+GitHub provides a useful notification feature for collaborative work - _mentions_.
+The mention system notifies team members when somebody else references them
+in an issue, comment or pull request -
+you can use this to notify people when you want to check a detail with them,
+or let them know something has been fixed or changed
+(much easier than writing out all the same information again in an email!).
+You can use the mention system to notify individual GitHub accounts
+or whole teams (when you need to mention multiple people at once and it is not practical to list them all individually).
+Typing `@` in GitHub will bring up a list of all accounts and teams linked to the repository that can be "mentioned".
```

Comment:
You can mention people who aren't linked with the repo already but it won't list them so you will need to know their github username

Reply:
Good point - will include this @sstevens2.